internal/summary/summary.go

```diff
@@ -91,14 +93,25 @@ func (s *Summary) Output(ctx context.Context, resolver LBResolver) string {
 		// K8s — assume that the first name will work
 		// TODO(han) - get the LoadBalancerSpec kind (ie: "service") dynamically
 		if len(svc.K8sLbs) > 0 {
-			ret += fmt.Sprintf("→ `kubectl get svc %s` ", svc.K8sLbs[0].Name)
+			lb := svc.K8sLbs[0]
+			namespaceFlag := ""
+			if lb.Namespace != "" {
+				namespaceFlag = fmt.Sprintf(" -n %s", lb.Namespace)
+			}
+
+			ret += fmt.Sprintf("→ `kubectl get svc %s%s` ", lb.Name, namespaceFlag)
 
 			url := resolver(ctx, svc.K8sLbs[0])
```

Comment:
...seems like the majority of this code can live in the same conditional, with this big being tacked on the end only if there exists a load balancer, no?

Reply:
not really, because it's parsing the data from different struct types